lib/src/fairing/fairings.rs

```diff
@@ -8,12 +8,17 @@ pub struct Fairings {
     launch: Vec<&'static Fairing>,
     request: Vec<&'static Fairing>,
     response: Vec<&'static Fairing>,
+    use_emoji: bool,
```

Comment:
This is strange at first glance -- ideally the Fairings struct would not need to know itself whether to print emojis or not, and it's another value to pass around and keep track of.